Packages/InputSystem/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "com.caseydecoder.unity-atoms-input-system",
```

Comment:
If it's supposed to be merged to the main project it should follow the same conventions as other packages, eg: `com.mambojambostudios.unity-atoms-input-system`.